_i18n/en/documentation/queue/lock.md

```diff
@@ -0,0 +1,7 @@
+A pretty common problem experienced by users is that they suddenly can not manually reorder the queue.
+
+If you can't move episodes around in the queue, this can be caused by either of the following:
+
+- `Keep Sorted` is enabled. You can disable this by tapping the three dots `…` in the top bar and select `Sort`.
+- The queue is locked. You can unlock the queue by tapping on the `unlock` icon in the top bar:
+  ![Unlocking the queue](/assets/images/documentation/{{ site.lang }}/queue-unlock.png)
```

Comment:
I would specify the language explicitly here. Currently, the image is missing when selecting German because the file is not translated and therefore also does not have a localized image.

Reply:
> Currently, the image is missing when selecting German because the file is not translated and therefore also does not have a localized image.

This we should fix by adding the translated image ;) (But sure, we can mention the language explicitly here - we just need to make sure that the translators are using the right language code then, though (URLs of links are included in PO files by mdpo).)